hdx_hapi/db/dao/util/util.py

```diff
@@ -65,22 +70,23 @@ class EntityWithLocationAdmin(Protocol):
 def apply_location_admin_filter(
     query: Select,
     db_class: Type[EntityWithLocationAdmin],
-    location_ref: Optional[int] = None,
-    location_code: Optional[str] = None,
-    location_name: Optional[str] = None,
+    common_location_parameters: CommonLocationParameters,
     has_hrp: Optional[bool] = None,
     in_gho: Optional[bool] = None,
```

Comment:
I'm wondering whether `has_hrp` and `in_gho` should go in one of these parameter clusters. I thought maybe CommonLocationParameters